charts/argo-workflows/templates/controller/workflow-controller-config-map.yaml

```
@@ -59,6 +59,9 @@ data:
       {{- if .Values.artifactRepository.gcs }}
       gcs:
 {{ toYaml .Values.artifactRepository.gcs | indent 8}}
+      {{- else if .Values.artifactRepository.azure }}
+      azure:
+{{ toYaml .Values.artifactRepository.azure | indent 8}}
```

```suggestion
      {{- with .Values.artifactRepository.azure }}
      azure:  {{ toYaml . | indent 8}}
      {{- end }}
```

much bigger fan of `with` statements over `if..else if..else` when using the value in the conditional.  Less `.Values...` text.  wdyt?

---

Thank you for reviewing!
I fixed on https://github.com/argoproj/argo-helm/pull/1488/commits/28418c7378e23b4ffacf01abebd77c33112a173e 🙋